packages/odyssey/src/scss/components/_tag.scss

```diff
@@ -17,11 +17,13 @@
   margin: 0 $spacing-xs $spacing-xs 0;
   padding: 0 $spacing-xs-em;
   border-radius: $base-border-radius;
-  background: cv('gray', '200');
+  background: cv('gray', '100');
   color: $text-body;
 }
 
 .ods-tag--list {
+  display: inline-block;
```

Comment:
I don't think we should change this to account for a horizontal label unless we're going to ship the label itself or include additional guidance around Parents. I'd prefer if we added a Horiz variant / horiz support separately.

Reply:
My understanding is that this is as @joejezowski-okta had intended for it to be by default, per the design. I will revert this for now and will schedule a follow-up chat/log an issue.